aip/_journals.tab

```diff
@@ -1,4 +1,6 @@
 Title	ISSN	eISSN	Field	Field2
+Journal of Vacuum Science &amp; Technology A: Vacuum, Surfaces, and Films 0734-2101 x physics chemistry
+Journal of Vacuum Science &amp; Technology B: Microelectronics and Nanometer Structures 1520-8567 x physics chemistry
```

Comment:
> I did use tabs for certain. not sure what happened.

Maybe a copy/paste problem? Some text editors also use "soft tabs" when you type a tab, which are just a certain number of spaces. You can usually change a preference in your text editor to show white space symbols so that you can tell spaces apart from tabs. (e.g. TextWrangler on macOS has an option "Show invisibles")